openqa-investigate

```diff
@@ -27,10 +29,19 @@ clone() {
         && echoerr "unable to clone job $id: it is part of a parallel or directly chained cluster (not supported)" && return 2
     name="$(echo "$job_data" | jq -r '.job.test'):investigate$name_suffix"
     base_prio=$(echo "$job_data" | jq -r '.job.priority')
+    clone_settings=("TEST=$name" '_GROUP_ID=0' 'BUILD=')
+    if [[ $refspec ]]; then
+        casedir=$(echo "$job_data" | jq -r '.job.settings.CASEDIR')
+        [[ $casedir == null ]] && casedir=''
+        repo=${casedir:-'https://github.com/os-autoinst/os-autoinst-distri-opensuse.git'}
+        clone_settings+=("CASEDIR=${repo%#*}#${refspec}")
+    fi
+    clone_settings+=("${@:4}")
     # clear "PUBLISH_" settings to avoid overriding production assets
     # shellcheck disable=SC2207
-    publish_keys=($(echo "$job_data" | jq -r '.job.settings | keys[] | select (startswith("PUBLISH_")) | . + "="'))
-    out=$($clone_call "$host_url/tests/$id" TEST="$name" _GROUP_ID=0 BUILD= "${publish_keys[@]}" "${@:3}")
+    clone_settings+=($(echo "$job_data" | jq -r '.job.settings | keys[] | select (startswith("PUBLISH_")) | . + "="'))
+    out=$($clone_call "$host_url/tests/$id" "${clone_settings[@]}")
+    [[ $dry_run = 1 ]] && echo "$out"
```

Comment:
are you abusing `$dry_run` as a debug flag? The call further down with `$client_call` should already provide the proper "dry-run" call. What you are interested here should be visible if you call the script with `-x` already

Reply:
`-x` makes the output *very* verbose. I want to show the clone invocation like the dry-run mode would actually do unless in this case because we capture the output here. The client call below doesn't show the arguments passed to the clone script.